### DISCUSSION TITLE
Support for Angular 18

### DISCUSSION THREAD

**Discussion by diparth1** [ng-maps/ng-maps#187] [Q&A] [answered]
Does ng-maps have support for Angular 18 or above versions?

I have dependency for @ng-maps/core, @ng-maps/google, and @ng-maps/places.

**Comment by DaSchTour** [comment]
Not actually through peerDependencies but it should work. I'll do the updates in the next days to have proper support.

**Reply by diparth1** [reply]
Can you provide an update with updated peerDependencies?

**Comment by DaSchTour** [comment, marked as answer]
So there should now be a version 6 with support for Angular 18 and a version 7 that supports Angular 19
But as I've seen the angular components version of maps has many new features. So I think that this may be the last version of ng-maps as I don't think that this library doesn't add any value over the implementation of the angular team.
